runtimes/peregrine/src/lib.rs

```diff
@@ -868,6 +905,9 @@ construct_runtime! {
 		// Preimage registrar
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 44,
 
+		// Tips module.
```

Comment:
Done in [31eaad3](https://github.com/KILTprotocol/mashnet-node/pull/352/commits/31eaad3e07e88c12967a808c06f56a8ac9129229). I was wondering whether we rather want to add the tips pallet to the Governance block though. One argument against this would be to leave space for Governance 2.0.